src/plugins/shared_ux/public/plugin.ts

```diff
@@ -37,3 +39,4 @@ export class SharedUXPlugin implements Plugin<SharedUXPluginSetup, SharedUXPlugi
 
   public stop() {}
 }
+export const getSharedUXServices = () => services;
```

Comment:
Yeah, we can _not_ do this.  This is exposing a stateful variable subject to a start contract in a module-level, stateless variable.